### PR TITLE
table made responsive

### PR DESCRIPTION
**Pull request by satyammittal**
- [X] The table of latest tests isn't any good on mobile (maybe we should use a responsive table plugin?)
In mobiles:
fca817de-6601-420a-98c8-e89ba188b6b9
In desktop:
b994581d-bcd2-47e9-a536-e4bd772bcd44
Table become list in small devices.
p.s.: sorry for the random data placed inside the table.
#35

### CODE REVIEW

**Comment by canihavesomecoffee** [comment]
Please update the PR to fix the aligment issues, and the "broken" html near the table element.

**Comment by satyammittal** [comment]
can you please specify what are the alignment issues?

**Comment by satyammittal** [comment]
screen shot 2017-02-15 at 16 37 19

File: templates/test/list_table.html [hunk 1]
```diff
@@ -1,26 +1,28 @@
...
-        <thead>
-        <tr>
-            <th>Repository</th>
-            <th>Details</th>
-            <th>Platform</th>
-            <th title="Indicates if an error occurred during any preparational step (merging, compiling).">Run errors</th>
-            <th title="Did the test suite finish or error out?">Completed</th>
+{% if tests|length > 0 %} 
+<div id="no-more-tables">
+<table class="col-md-12 table-bordered table-striped table-condensed cf">
```
Comment (canihavesomecoffee) [inline templates/test/list_table.html]: Alignment here.

Comment (satyammittal) [inline templates/test/list_table.html]: ok

**Comment by canihavesomecoffee** [comment]
That's looking all good, but could you now fix the conflict between the master and your PR in app.css?

**Comment by satyammittal** [comment]
ok i will fix the conflict

**Comment by madan96** [comment]
@satyammittal Try fetch and rebase instead of pull. It results in a cleaner commit history :) . Also, I think you can `squash` the many of the commits.

**Comment by canihavesomecoffee** [comment]
I definitely am going to squash that :p

**Pull request merged**